certified-connectors/RAPID Platform/readme.md

```diff
@@ -0,0 +1,44 @@
+## RAPID Platform Connector
+The RAPID Platform provides a highly customisable data creation and tracking tools for business. Using this API you can automate business logic based on your own specific needs.
+
+
```

Comment:
@MattFranklin-Rapid Extra spacing can be removed

Reply:
Removed spacing in readme to make consistent one line space before headings